cmake/cc_rules.cmake

```diff
@@ -112,24 +112,56 @@ function(cc_test name)
   target_link_libraries(${name} ${cct_DEPENDS})
 endfunction()
 
-# cc_binary(
+# cc_fuzz_test(
```

Comment:
Optional: You might consider breaking this out into a fuzzing specific cmake library, especially if you're going to add more fuzzing-specific cmake code in here. If not, feel free to leave as is.

Reply:
I do not expect to add any more (cmake or other) code in here. Will leave it as is for now. Thanks